+hw/+ptb/Window.m

```diff
@@ -612,14 +612,32 @@ function applyCalibration(obj, cal)
       Screen('LoadNormalizedGammaTable', obj.PtbHandle, gammaTable);
     end
     
-    function c = calibration(obj, dev, lightIn, clockIn, clockOut)
+    function c = calibration(obj, dev, lightIn, clockIn, clockOut, makePlot)
```

Comment:
how would you feel about something like this?
```
      function c = calibration(obj, dev, lightIn, varargin)
      %  Creates a file that specifies gamma calibration for a screen.
      %  This function requires a user to hold a photodiode, connected to a NI-DAQ, against a screen in order
      %  to perform gamma calibration, and save the calibration settings to a file.
      %
      %  Inputs:
      %    dev (int) : NI DAQ device ID to which the photodiode is
      %      connected
      %    lightIn (char) : analogue input channel name to which the
      %      photodiode is connected
      % 
      %  Optional Inputs:
      %    clockIn (char) : analogue input channel name for clocking pulse
      %    clockOut (char) : digital output channel name for clocking pulse
      %    makePlot (bool) : flag for making photodiode signal plot
      %
      %  Outputs:
      %    c (struct) : calibration struct containing refresh rate and
      %      gamma tables
      %
      %  Examples:
      %
      % See also: `calibrationStruct`
```
I think having varargin and those optional inputs is clarifying here, and then an args struct could be used to specify default input args in the function body, which I like better than the `if nargin < n`

Reply:
fine